### DISCUSSION TITLE
Possible to edit other people's comments (only as admin?)

### DISCUSSION THREAD

**Discussion by lapinskap** [community/community#61607] [Repositories]
### Select Topic Area

Bug

### Body

Possible to edit other people's comments in MR even without any trace - strage bug spotted this morning. I'm able to edit whole content of any comment, without the other person knowing. Tested few private repos that I have full rights in. Even as a admin, it is not cool to change what someone else is saying.

Edit 25.07.2023r.:
Seems like it is possible to edit comments without admin rights, as all team members can do that, in various projects

Steps to reproduce:
1. Create merge request
2.  receive comment in MR from another account
3. Click on tree dots in top right corner of the comment
4. click "Edit"
5. Edit comment, click save

![MicrosoftTeams-image (4)](https://github.com/community/community/assets/29812278/2ff9cddb-7f17-40ad-8d66-7b7e1a9c69c1)

![MicrosoftTeams-image (5)](https://github.com/community/community/assets/29812278/ca3147f7-8164-4730-bb6c-3bdf7a631035)

**Comment by PaulinaLapinska** [comment]
Edit resolved comment, from someone else, who is author of their merge request, in which you didn't participate at all? No problem!

![Screenshot_1](https://github.com/community/community/assets/123079983/756d85ba-d093-4cc9-8267-f7835fc8e063)

![Capture2](https://github.com/community/community/assets/123079983/81d8ef9c-b16f-4e35-9a3f-90f616027d1a)

Even if you're the admin of repo, I think you shouldn't be able to edit every comment, rather just delete.

**Reply by lapinskap** [reply]
Here, on airbnb: https://github.com/airbnb/javascript/pull/2757 we can't edit random comments, so likely it is only for admin rights in repo

**Comment by xaedes** [comment]
Agreed, it should not be possible to edit others' comments. I want to least be able to disable that ability so that I can't (accidentally) do it.
